docs/victoriametrics/Cluster-VictoriaMetrics.md

```diff
@@ -247,6 +247,8 @@ If you need multi-AZ setup, then it is recommended running independent clusters
 into all the cluster - see [these docs](https://docs.victoriametrics.com/vmagent/#multitenancy) for details.
 Then an additional `vmselect` nodes can be configured for reading the data from multiple clusters according to [these docs](#multi-level-cluster-setup).
 
+See [victoria-metrics-distributed chart](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-distributed) for an example.
```

Comment:
it is better to use https://docs.victoriametrics.com/helm/victoriametrics-cluster/